isis/src/galileo/apps/gllssical/main.cpp

```diff
@@ -440,11 +440,18 @@ void calculateScaleFactor0(Cube *icube, Cube *gaincube) {
     Spice spicegll(*icube);
     spicegll.instrumentPosition()->SetAberrationCorrection("LT+S");
     QString startTime = label->findGroup("Instrument",Pvl::Traverse)["SpacecraftClockStartCount"][0];
-    Isis::FileName sclk(label->findGroup("Kernels",Pvl::Traverse)["SpacecraftClock"][0]);
-    QString sclkName(sclk.expanded());
-    furnsh_c(sclkName.toLatin1().data());
     double obsStartTime;
-    scs2e_c(-77, startTime.toLatin1().data(), &obsStartTime);
+
+    try {
+      obsStartTime = spicegll.getClockTime(startTime.toLatin1().data(), -77).Et();      
+    } 
+    catch (IException &e) {
+      Isis::FileName sclk(label->findGroup("Kernels", Pvl::Traverse)["SpacecraftClock"][0]); 
+      QString sclkName(sclk.expanded()); 
+      furnsh_c(sclkName.toLatin1().data());
```

Comment:
This needs a naif check around it with a catch. If the furnish fails everything after that will be bad.